### ISSUE TITLE
Enabler: Review Factbook Highcharts for Accessibility Compliance

### ISSUE THREAD

**Issue by laurelthrash** [NCIOCPL/cgov-digital-platform#4292]
### Description
During preparation for the Factbook FY23 update, the team noted that it was uncertain if the Factbook high charts have been reviewed for accessibility compliance. The understanding is that there are accessibility features built into the highchart code but this is not certain and should be verified. Request is to investigate this to validate that the highcharts meet 508 compliance on Cancer.gov. 

### Resources:
* Feature: N/A
* Mockups: N/A
* References: N/A
* Web Page: https://www.cancer.gov/about-nci/budget/fact-book

### Prerequisites
- N/A

### Sub-Tasks
- [ ] TBD
- [ ] TBD

### Notes
- N/A
